src/lib/dates.test.jsx

```diff
@@ -0,0 +1,40 @@
+import { describe, it, expect, vi } from "vitest";
+import { getDayOfWeek } from "./dates";
+
+describe("getDayOfWeek", () => {
+  it("should return 'Tomorrow' for the date after today", () => {
+    const mockDate = new Date(2024, 0, 7);
+    vi.setSystemTime(mockDate);
+
+    const dayIndex = (mockDate.getDay() + 1) % 7;
+    expect(getDayOfWeek(dayIndex)).toBe("Tomorrow");
+  });
```

Comment:
Inside tests we try to avoid logic as much as possible, so you should use this calculation of module or anything else (doing that, you're doing the code)

```suggestion
  it("should return 'Tomorrow' for the date after today", () => {
    const today = new Date(2024, 0, 7);
    vi.setSystemTime(mockDate);

    const tomorrow = new Date(2024, 0, 8).getDay();
    expect(getDayOfWeek(tomorrow)).toBe("Tomorrow");
  });
```